.github/workflows/publish_nuget.yml

```diff
@@ -13,7 +13,13 @@ jobs:
           dotnet-version: |
             7.0.304
             6.0.300
-      - run: dotnet test Src/CSharpier.Tests/CSharpier.Tests.csproj
+      - run: >
+          dotnet test Src/CSharpier.Tests/CSharpier.Tests.csproj
+          --configuration Release
+          --logger "GitHubActions;summary.includePassedTests=true;summary.includeSkippedTests=true"
+          --
+          RunConfiguration.CollectSourceInformation=true
+
```

Comment:
I added `--configuration Release` here because it looks like it was unintentionally left out